denizen_scripts/survival/repo_link/systems/vote_handlers.dsc

```diff
@@ -0,0 +1,151 @@
+vote_reciever:
+  type: world
+  debug: false
+  events:
+    #Vote Recieved:
+      on votifier vote:
+      - define listed_sites <element[to_be_filled_in_when_listed_on_sites]>
+      - define voter <server.match_offline_player[<context.username>]||invalid>
+      - if <[voter]> != invalid:
+        - inject daily_vote_counter
+        - inject weekly_vote_counter
+  #Check for offline keys and award/announce
+      on player joins:
+      - if <player.flag[weekly_crate_pending]>:
+        - narrate: "<&e>You have a <item[weekly_vote_key].display> <&e>pending. Do <&3>/<&b>weeklykeys <&e>to claim it.!"
+      - if <player.flag[enderchest_daily_key_offline]> > 0:
+        - if <player.flag[enderchest_daily_key_offline]> > 1:
+        - narrate "<&e>Your inventory was full and <&a><player.flag[enderchest_daily_key_offline]> <item[daily_vote_key].display>s<&e> were deposited into your enderchest!"
+      - else:
+        - narrate "<&e>Your inventory was full and <&a>a <item[daily_vote_key].display>s<&e> was deposited into your enderchest!"
+      - if <player.flag[inventory_daily_key_offline]> > 0:
+        - if <player.flag[inventory_daily_key_offline]> > 1:
+        - narrate "<&e>You recieved <player.flag[inventory_daily_key]> <item[daily_vote_key].display>s<&e> while offline!"
+      - else:
+        - narrate "<&e>You recieved <&a>a <item[daily_vote_key].display><&e> while you were offline!"
+      - if <player.flag[daily_key_pending]> > 0:
+        - if <player.flag[daily_key_pending]> > 1:
+          - narrate "<&c>You have <player.flag[daily_key_pending]> <item[daily_vote_key].display>s<&c> waiting for you at <&3>/<&b>recoverkeys."
+        - else:
+          - narrate "<&c>You have a <item[daily_vote_key].display><&c> waiting for you at <&3>/<&b>recoverkeys."
+      - if <player.flag[enderchest_daily_key_offline].add[<player.flag[inventory_daily_key_offline]>]> > 0:
+        - if <player.flag[enderchest_daily_key_offline].add[<player.flag[inventory_daily_key_offline]>]> > 1:
+          - announce "<&c><player.display_name> has recieved <&a><player.flag[enderchest_daily_key_offline].add[<player.flag[inventory_daily_key]>]> <item[daily_vote_key].display>s<&c> for voting while offline!<&nl><&c>Do <&3>/<&b>vote<&c> to recieve yours now!"
+        - else:
+          - announce "<&a><player.display_name> <&c>has just recieved a <item[daily_vote_key].display><&c>! Do <&3>/<&b>vote<&c> to recieve yours now!""
+
+
+vote_handler:
+  type: task
+  debug: false
+  daily_vote_counter:
```

Comment:
this script container is missing the `script` key; if it isn't really needed to be separated from the script that injects it, you could move this to a local key and inject it locally, elsewise you could move the script in `daily_vote_counter` into the `script` key and inject the script as is

Reply:
I put this separate because I wasn't aware i could put a `script:` in a `type: world` container. Do I just add `script:` as I would for `events:`?